### PR TITLE
[V6] Add ERC20 incentive type, improve multiplier

### PR DESCRIPTION
**Pull request by brkagithub**

### CODE REVIEW

File: dkg/paranet.py [hunk 1]
```diff
@@ -259,8 +265,13 @@ def is_voter(
         self,
         ual: UAL,
         address: Address | None = None,
-        incentives_type: ParanetIncentivizationType = ParanetIncentivizationType.NEUROWEB,
```
Comment (u-hubar) [inline dkg/paranet.py:262]: This is a breaking change, we shouldn't do that imo

Comment (u-hubar) [reply dkg/paranet.py:262]: I c your logic, but I would rather make default None and if it's None, then figure it out from the ual

File: dkg/paranet.py [hunk 2]
```diff
@@ -457,11 +508,9 @@ def update_claimable_rewards(self, ual: UAL) -> dict[str, str | HexStr | TxRecei
         }
 
     def _get_incentives_pool_contract(
-        self,
-        ual: UAL,
-        incentives_type: ParanetIncentivizationType = ParanetIncentivizationType.NEUROWEB,
+        self, ual: UAL, incentives_type: ParanetIncentivizationType
     ) -> str | dict[str, str]:
-        incentives_pool_name = f"Paranet{str(incentives_type)}IncentivesPool"
+        incentives_pool_name = f"ParanetNeurowebIncentivesPool"
```
Comment (u-hubar) [inline dkg/paranet.py:513]: Let's move it to constants perhaps

File: dkg/dataclasses.py [hunk 3]
```diff
@@ -55,8 +55,9 @@ class KnowledgeAssetContentVisibility(AutoStrEnumUpperCase):
     PRIVATE = auto()
 
 
-class ParanetIncentivizationType(AutoStrEnumCapitalize):
-    NEUROWEB = auto()
+class ParanetIncentivizationType(Enum):
+    NEUROWEB = "Neuroweb"
+    ERC20 = "NeurowebERC20"
```
Comment (u-hubar) [inline dkg/dataclasses.py:60]: Should we call it `NEUROWEB_ERC20`? There is no support for any custom ERC20 atm, so perhaps it would be more clear

**Review by br41nlet — CHANGES_REQUESTED** [review]
Some minor adjustments needed. Also don't forget to bump the package version in the config

File: dkg/paranet.py [hunk 4]
```diff
@@ -91,7 +98,7 @@ def deploy_incentives_contract(
         self,
         ual: UAL,
         incentives_pool_parameters: NeuroWebIncentivesPoolParams,
-        incentives_type: ParanetIncentivizationType = ParanetIncentivizationType.NEUROWEB,
```
Comment (u-hubar) [inline dkg/paranet.py:94]: Let's not remove default

**Review by br41nlet — CHANGES_REQUESTED** [review]

**Review by br41nlet — APPROVED** [review]

**Pull request merged**